### PR TITLE
R pi4 install

### PR DESCRIPTION
**Pull request by lightninginabox**
Update Raspberry Pi Instructions and added installation scripts for LND & CLightning

### CODE REVIEW

**Review by Zaxounette — CHANGES_REQUESTED** [review]
ACK grammar, format and such.

But you created .txt files.
@dennisreimann , are TXT files okay? I'd prefer having MD files all around, but if they're not a problem for deploying to the doc website...

**Review by britttttk — APPROVED** [review]

**Review by Zaxounette — APPROVED** [review]

**Comment by NicolasDorier** [comment]
The scritps referenced do not exists

File: docs/Deployment/RPi4.md [hunk 1]
```diff
@@ -2,8 +2,47 @@
...
+
+Change your password. 
+```bash
+passwd
+```
+
+For LND Full Node - IBD takes approximately 36 hours after initial install.
+```bash
+sudo su -
+wget -O btcpayserver-install-lnd.sh https://raw.githubusercontent.com/btcpayserver/btcpayserver-doc/master/docs/Deployment/btcpayserver-lnd-rpi4-install.md
```
Comment (NicolasDorier) [inline docs/Deployment/RPi4.md:30]: wat? you can execute a md file?

File: docs/Deployment/RPi4.md [hunk 2]
```diff
@@ -2,8 +2,47 @@
...
+chmod +x btcpayserver-install-lnd.sh
+. btcpayserver-install-lnd.sh
+```
+
+For C-Lightning Full Node - IBD takes approximately 36 hours after initial install.
+```bash
+sudo su -
+wget -O btcpayserver-install-clightning.sh https://raw.githubusercontent.com/btcpayserver/btcpayserver-doc/master/docs/Deployment/btcpayserver-clightning-rpi4-install.md
```
Comment (NicolasDorier) [inline docs/Deployment/RPi4.md:38]: I don't execute md files!

**Comment by NicolasDorier** [comment]
Even if they existed, I am wondering if we should provide at all the ready to run scripts.

**Comment by lightninginabox** [comment]
The install scripts have .md extensions on GH, but the "wget" command renames them to .sh. They will exist on your repo if the PR gets merged. 

"**wget -O btcpayserver-install-clightning.sh** https://raw.githubusercontent.com/btcpayserver/btcpayserver-doc/master/docs/Deployment/btcpayserver-clightning-rpi4-install.md"

In my experience selling hardware nodes most people just want to get a "full node" up and running as quickly as possible and they work out the details later (Tor vs clear net, domain name, port forwarding etc.)  The benefit to having an install script it that saves the user a lot of copy & pasting from the instructions.

**Comment by pavlenex** [comment]
> Even if they existed, I am wondering if we should provide at all the ready to run scripts.

This is just first step. First we create easy to run scripts, next step is to provide option for an image which would deploy Transmuter for people who need customisation. The reason Umbrel-like services are popular is because all of the pre-configured things. The simpler and easier the more people will opt in.

**Pull request merged**